examples/utils.py

```
@@ -23,13 +23,80 @@
+    def __init__(self,
+                 vocab_file: str,
+                 padding_side: str = 'left',
+                 truncation_side: str = 'left'):
+        import sentencepiece as spm
+        sp = spm.SentencePieceProcessor()
+        sp.Load(vocab_file)
+        self.sp_model = sp
+        self.padding_side = padding_side
+        self.truncation_side = truncation_side
+        self.vocab_size = sp.GetPieceSize()
+        _opt = lambda i: i if i >= 0 else None
```

Ruff (0.15.12)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `_opt` as a `def`

(E731)